src/jit/lowerxarch.cpp

```diff
@@ -2901,6 +2901,12 @@ bool Lowering::IsContainableHWIntrinsicOp(GenTreeHWIntrinsic* containingNode, Ge
             return supportsUnalignedSIMDLoads;
         }
 
+        case NI_AVX_ExtractVector128:
```

Comment:
Is this independent JIT bug fix? It may be best to submit this JIT bug fix as separate PR, so that it gets the right level of scrutiny as other JIT bug fixes.

Reply:
Yes, it's just avoiding the assert from the default case.

Will extract.

Reply:
Extracted to https://github.com/dotnet/coreclr/pull/27807